index.js

```diff
@@ -294,6 +304,12 @@ export default class WebTorrent extends EventEmitter {
     return torrent
   }
 
+  addV2 (torrentId, timeout) {
```

Comment:
May be better to provide comment & typings

Reply:
ptal